salt/kubelet/kubelet.jinja

```diff
@@ -7,51 +7,46 @@
 # kubernetes kubelet (minion) config
 
 # The address for the info server to serve on (set to 0.0.0.0 or "" for all interfaces)
-KUBELET_ADDRESS="--address=0.0.0.0"
+# DEPRECATED!!! This option was moved to kubelet-config.yaml
+KUBELET_ADDRESS=""
 
 # The port for the info server to serve on
-KUBELET_PORT="--port={{ pillar['kubelet']['port'] }}"
+# DEPRECATED!!! This option was moved to kubelet-config.yaml
+KUBELET_PORT=""
 
 # Ensure we match the machine hostname
 KUBELET_HOSTNAME="--hostname-override={{ grains['nodename'] }}"
 
 # Add your own!
 KUBELET_ARGS="\
-    --cgroups-per-qos \
-    --cgroup-driver=cgroupfs \
-    --cgroup-root=/ \
-    --kube-reserved-cgroup=podruntime.slice \
-{% if salt.caasp_pillar.get_kubelet_reserved_resources('kube') -%}
+    --config=/etc/kubernetes/kubelet-config.yaml \
+{%- if salt.caasp_pillar.get_kubelet_reserved_resources('kube') %}
     --kube-reserved={{ salt.caasp_pillar.get_kubelet_reserved_resources('kube') }} \
-{% endif -%}
-    --system-reserved-cgroup=system \
-{% if salt.caasp_pillar.get_kubelet_reserved_resources('system') -%}
+{%- endif %}
+{%- if salt.caasp_pillar.get_kubelet_reserved_resources('system') %}
     --kube-reserved={{ salt.caasp_pillar.get_kubelet_reserved_resources('system') }} \
-{% endif -%}
-    --enforce-node-allocatable=pods \
-{% if pillar['kubelet']['eviction-hard'] -%}
+{%- endif %}
+{%- if pillar['kubelet']['eviction-hard'] %}
     --eviction-hard={{ pillar['kubelet']['eviction-hard'] }} \
```

Comment:
`eviction*` also [appear in the struct](https://github.com/kubernetes/kubernetes/blob/release-1.10/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go#L324)... could it be moved?

Reply:
Ok, I will move it in another PR, since these flags are not deprecated yet